src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
1. The  `people.filter((p) => p.sex === 'm')` is duplicated. 
2. It's better to name object with full name `p` -> `person` or even better to do destructuring

So I would recommend to do something like this:

`const men = people.filter((person) => person.sex === 'm' && (!century || Math.ceil(person.died / 100) === century))`

Or

`const men = people.filter(({ sex, died }) => sex === 'm' && (!century || Math.ceil(died / 100) === century))`

Expl.: if `century` has falsy value than `!century` return `true `and `Math.ceil(p.died / 100) === century` won't be executed and only ` p.sex === 'm'` condition will be working. 

But if `century` is a number then `!century` return `false` and `Math.ceil(p.died / 100) === century` will be executed as well. And full condition will looks like `p.sex === 'm' && Math.ceil(p.died / 100) === century`